jobs/generation/MetaGenerator.groovy

```diff
@@ -227,11 +227,13 @@ repos.each { repoInfo ->
         return;
     }
 
-    // Make the folders
+    // Make the folders. Save the root folder for overview generation
     def generatorFolder = ''
+    def projectFolder = ''
     for (folderElement in repoInfo.folders) {
         if (generatorFolder == '') {
             generatorFolder = folderElement
+            projectFolder = folderElement
```

Comment:
The private folder will need a separate PR in dotnet-ci-internal, and I was planning on handling it to look for the first folder that isn't named "Private". If you look at the example dashboard I have in roslyn, it references all jobs by their paths, so duplicates are disambiguated. Screenshot:
![image](https://cloud.githubusercontent.com/assets/2371880/24672660/64b0ae2e-192a-11e7-9503-490373fe2c1f.png)